Contributing revamp, take 2 (#10956)
netdata/netdata: README.md

## Menu

> - [Menu](#menu)

Reviewer: I don't think we need this very self-referential item.

## Community

> Netdata is striving to be an all inclusive open-source project and community. Please read our [Code of Conduct](https://learn.netdata.cloud/contribute/code-of-conduct).

Reviewer: Let's be a bit more confident here.

```suggestion
Netdata is an inclusive open-source project and community. Please read our [Code of Conduct](https://learn.netdata.cloud/contribute/code-of-conduct).
```

> You can find most of the Netdata team in our [community forums](https://community.netdata.cloud). It's the best place to

Reviewer: ```suggestion
Find most of the Netdata team in our [community forums](https://community.netdata.cloud). It's the best place to
```

## Contribute

> Contributions are the lifeblood of OSS projects, such as Netdata Agent. While we are continuing to invest resources and improve the OSS Agent, we need every help we can get in our effort to democratize monitoring!

Reviewer: ```suggestion
Contributions are the lifeblood of open-source projects. While we continue to invest in and improve Netdata, we need help to democratize monitoring!
```

> - Read our [Contributing Guide](https://learn.netdata.cloud/contribute/handbook). In there, you will find all the information you need to contribute to the Netdata Agent, from improving our documentation to engaging in the community and development. We have made efforts to make it as frictionless as possible, but if you need help, just ping us on our community forums!
> - We have a whole category dedicated to contributing and extending the Netdata Agent on our [community forums](https://community.netdata.cloud/c/agent-development/9)
> - You have found a bug? Open a [GitHub issue](https://github.com/netdata/netdata/issues/new?assignees=&labels=bug%2C+needs+triage&template=bug_report.md).
> - View our [Security Policy](https://github.com/netdata/netdata/security/policy)

Reviewer: ```suggestion
- Read our [Contributing Guide](https://learn.netdata.cloud/contribute/handbook), which contains all the information you need to contribute to Netdata, such as improving our documentation, engaging in the community, and developing new features. We've made it as frictionless as possible, but if you need help, just ping us on our community forums!
- We have a whole category dedicated to contributing and extending Netdata on our [community forums](https://community.netdata.cloud/c/agent-development/9)
- Found a bug? Open a [GitHub issue](https://github.com/netdata/netdata/issues/new?assignees=&labels=bug%2C+needs+triage&template=bug_report.md).
- View our [Security Policy](https://github.com/netdata/netdata/security/policy).
```
